FormlyMapper/FormlyField/FileField.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Linio\DynamicFormBundle\FormlyMapper\FormlyField;
+
+use Linio\DynamicFormBundle\FormlyMapper\FormlyField;
+
+class FileField extends FormlyField
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getFieldType()
+    {
+        return 'file';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function buildFieldTypeConfiguration()
+    {
+        $this->formlyFieldConfiguration['type'] = $this->getFieldType();
```

Comment:
Yes because into `FormlyField.php` i'm setting this key `$this->formlyFieldConfiguration['templateOptions']['type']` and here overrides this other one `$this->formlyFieldConfiguration['type']` declared into `FormlyField.php` like `input`.